OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/Optimove.java

```diff
@@ -143,29 +146,73 @@ public static void initialize(@NonNull Application application, @NonNull Optimov
             Optimobile.initialize(application, config, shared.userInfo.getInitialVisitorId(), shared.userInfo.getUserId());
         }
 
-        if (config.isOptimoveConfigured()) {
+        if (config.isOptimoveConfigured()){
```

Comment:
I think the whole file misses the format

Reply:
was caring about reviewers again planning to do formatting in the end 😢 updated